Smartling.API/Model/JobItem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Newtonsoft.Json;
+
+namespace Smartling.Api.Model
+{
+    public class JobItem
```

Comment:
This name confuses because of we have Jobs API and everyone who will use this SDK will be sure that this class is related to that API. I don't have better name and can only propose use another namespace.
* Put this and other "response" related files into `...Model.TranslationDashboard` folder
* Update their namespaces

Reply:
Done.